### DISCUSSION TITLE
taskbar-clock-customization make it to show 24Hour

### DISCUSSION THREAD

**Discussion by hendrikbez** [ramensoftware/windhawk-mods#630] [Q&A]
Hoe cam I make the taskbar-clock-customization to show in 24hour and not AM or PM

**Comment by m417z** [comment]
Refer to [the time format](https://learn.microsoft.com/en-us/windows/win32/api/datetimeapi/nf-datetimeapi-gettimeformatex#remarks). You can replace the default time format, `hh':'mm':'ss tt`, with `HH':'mm':'ss`.

**Comment by hendrikbez** [comment]
Thank you
